src/components/DelegateStatement/NotificationSelector.tsx

```diff
@@ -0,0 +1,62 @@
+import { Tabs, TabsList, TabsTrigger } from "@/components/ui/tabs";
+import { type UseFormReturn } from "react-hook-form";
+import { type DelegateStatementFormValues } from "./CurrentDelegateStatement";
+import { useState, useEffect } from "react";
+
+export default function NotificationSelector({
+  form,
+}: {
+  form: UseFormReturn<DelegateStatementFormValues>;
+}) {
+  const [subscribed, setSubscribed] = useState(false);
+
+  useEffect(() => {
+    setSubscribed(
+      form.getValues("notificationPreferences.wants_proposal_created_email") ||
+        form.getValues(
+          "notificationPreferences.wants_proposal_ending_soon_email"
+        )
+    );
```

Comment:
if either of the values are true, set the whole object to true (we aren't yet splitting this up in the UI, maybe we should?)